src/user_templates_api/templates/jupyter_lab/templates/hra_api_tutorial/template.txt

```diff
@@ -0,0 +1,1283 @@
+      "    sex - (string)  (optional) (male, female, both)\n",
+      "    technologies - (string) (optional) (List of technologies - api_instance.technology_names)\n",
+      "    providers - (string) (optional) \n",
+      "    ontology_terms - (string) (optional) \n",
+      "    cell_type_terms - (string) (optional) \n",
+      "    spatial - SpatialSearch(x, y, z, radius, target) (optional) \n",
+      "    token - (string) (optional) \n",
+      "</pre>"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "id": "db1e1ebd",
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "age = MinMax(min=45.0, max=60.0)\n",
+      "bmi = MinMax(min=65.0, max=83.0)\n",
+      "cache = True\n",
+      "sex = \"female\"\n",
+      "technologies = []\n",
+      "providers = []\n",
+      "ontology_terms = [\"http://purl.obolibrary.org/obo/UBERON_0000955\",]\n",
+      "cell_type_terms = [\"http://purl.obolibrary.org/obo/CL_0000000\",] \n",
+      "spatial = [SpatialSearch(\n",
+      "            x=84.0,\n",
+      "            y=146.0,\n",
+      "            z=53.0,\n",
+      "            radius=12.0,\n",
+      "            target=\"http://purl.org/ccf/latest/ccf.owl#VHFAllenBrain\",\n",
+      "        ),]\n",
+      "token = \"\"\n",
+      "\n",
+      "try:\n",
+      "    api_response = api_instance.cell_type_term_occurences(age=age, bmi=bmi, cache=cache, ontology_terms=ontology_terms, cell_type_terms=cell_type_terms, providers=providers, sex=sex, spatial=spatial, technologies=technologies, token=token)\n",
+      "    pprint(api_response)\n",
+      "except hra_api_client.ApiException as e:\n",
+      "    print(\"Exception when calling DefaultApi->aggregate_results: %s\\n\" % e)"
+    ]
+  },
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "0e8bd39d",
+    "metadata": {},
+    "source": [
+      "***\n",
+      "<a id='ontology_nodes'></a>\n",
+      "#### Get ontology term tree nodes\n",
+      "<pre>\n",
+      "Accepted Parameters:\n",
+      "    cache - (boolean) (optional)\n",
+      "    token - (string) (optional)\n",
+      "</pre>"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "id": "7d5e5dd0",
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "try:\n",
+      "    api_response = api_instance.ontology_tree_model(cache=True)\n",
+      "    for item in api_response:\n",
+      "        pprint(item)\n",
+      "except hra_api_client.ApiException as e:\n",
+      "    print(\"Exception when calling DefaultApi->aggregate_results: %s\\n\" % e)"
+    ]
+  },
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "7170f6f3",
+    "metadata": {},
+    "source": [
+      "***\n",
+      "<a id='cell_nodes'></a>\n",
+      "#### Get cell type tree nodes\n",
+      "<pre>\n",
+      "Accepted Parameters:\n",
+      "    cache - (boolean) (optional)\n",
+      "    token - (string) (optional)\n",
+      "</pre>"
+    ]
+  },
```

Comment:
This cell produces ~1000 lines. See the other comments for cells that produce large outputs